service_configuration_lib/spark_config.py

```diff
@@ -409,7 +432,7 @@ def _get_k8s_spark_env(
     paasta_pool: str,
 ) -> Dict[str, str]:
     spark_env = {
-        'spark.master': f'k8s://https://k8s.paasta-{paasta_cluster}.yelp:16443',
+        'spark.master': f'k8s://https://k8s.{paasta_cluster}.paasta:6443',
```

Comment:
I also created the PR for [this](https://github.com/Yelp/service_configuration_lib/pull/65). 
It's good it's been done as part of single PR, I will discard mine.